docker/Dockerfile.dev

```diff
@@ -29,7 +29,7 @@ RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build \
 ############################
 # using static nonroot image
 # user:group is nonroot:nonroot, uid:gid = 65532:65532

```

Comment:
Could you modify the comment, so that this won't happen in the future? Also "static" is wrong now :D